contracts/staking/StakingInfo.sol

```diff
@@ -99,6 +99,11 @@ contract StakingInfo {
         uint256 indexed rewards,
         uint256 tokens
     );
+    event DelReStaked(
```

Comment:
Not clear from the name what this event does.

Reply:
Del is a prefix for delegation related events and `ReStaked` is for `reStaking`.

Reply:
Clearer name would have helped. Approving...